tests/test_hooks.py

```diff
@@ -54,6 +54,20 @@ def test_get_hook_caller(simple_plugin):
     assert friction_factor(1, 2) == 3
 
 
+def test_get_hook_caller_passing_ignored_plugins(datadir, simple_plugin, simple_plugin_2):
+    plugins_dirs = [simple_plugin['path'], simple_plugin_2['path']]
+    hm = HookMan(specs=simple_plugin['specs'], plugin_dirs=plugins_dirs)
+
+    assert len(hm.get_plugins_available()) == 2
+    assert len(list((datadir / 'plugins').iterdir())) == 2
+
+    hook_caller = hm.get_hook_caller(ignored_plugins=['Simple Plugin 2'])
```

Comment:
Hmm by caption? We should ignore by some identifier, not the caption...

Reply:
I will do this in another PullRequest.

Reply:
https://github.com/ESSS/hookman/issues/26